scripts/irods/lib.py

```diff
@@ -786,6 +786,19 @@ def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
 
             bytes_written += to_write
 
+def create_ascii_printable_file(filename, filesize):
+    """Creates a file with pseudo-random ASCII printable characters.
+    
+    - Does NOT use OS entropy (no `os.urandom` or `secrets`).
+    - Uses `random` for deterministic pseudo-randomness.
+    """
```

Comment:
I think we should seed the pseudo-randomizer like in `make_arbitrary_file(...)`. (i.e., `random.seed(5)`)

Otherwise, we end up with a pseudo-randomizer with an unknown start state. This would mean the state would now depend on the order the tests are executed and the number of parallel test containers used.

Reply:
Will add a parameter to allow the caller to set a seed.

Reply:
Done.